content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,452 @@
+
+## An adjustable histogram and a new mode: RGB parade
+
+Two new features in darktable 3.2 histogram. First is the possibility to adjust histogram height size. This adjustment is done the same way as some lighttable modules: move the cursor over the histogram and press `Ctrl` + `scroll` to adjust the height.
+
+A new mode, RGB parade, appears and displays waveforms that represent the levels of each layer (red, green and blue). This mode allows you to visualize the distribution of color components.
+
+This mode can also be displayed by selecting the histogram display in wave mode (with the top left icon on the histogram) and then  changing the wave mode with the next icon on the right.
+
+  <div style="text-align: center">
+![RGB parade mode]({filename}en/rgb-parade.png)
+  </div>
+
+Add to that an important code rewriting of the histogram for better performances.
+
+## Drawn mask: curved gradient
+
+When adding a drawn mask, it is now possible to bend the gradient shape (gradient). To do this, point the mouse cursor at the gradient bar and scroll up or down with the mouse wheel (or touchpad). `Ctrl` + `scroll` allows you to set the opacity and `Shift` + `scroll` manages the edge hardness.
+
+## An improved history
+
+The history behaviour has been improved according to different users feedback. When compressing the history or loading a new image, the history will now behave this way:
+
+- any active module that cannot be deactivated will always appear first in the history stack;
+- any module that is enabled by default but can be disabled appears in the history stack;
+- finally, all modules activated by presets or as a result of a preference setting (workflow or sharpness selected in the preferences) are added.
+
+## And more:
+
+- Add support of AVIF file format (>= 0.7). This require the [libavif](https://github.com/AOMediaCodec/libavif) library to be installed.
+- Add optional grey-scale export of TIFF for monochrome images.
+- It is now possible to export masks in TIFF format.
+- It is now possible to use more than 500 images in tethered control, which is needed for time-lapse.
+- Applying a style now supports _append_ mode (added to the existing image history) or in _overwrite_ mode (so replace the existing image history). This makes the style module consistent with _development_ module.
+- It is possible to handle (deleting, applying, or exporting) multiple styles in the style module.
+- Implement undo/redo for orientation changes from the lighttable.
+- By using `Ctrl` + `click` on mask shapes buttons, you can continuously create similar shapes.
+- Rejecting an image keeps in memory the previously defined number of stars. If the rejection is canceled, the image will be returned to the defined number of stars.
+
+This list is not exhaustive. Some other minor improvements could be discovered by reading the complete darktable release notes: https://github.com/darktable-org/darktable/releases/tag/release-3.2.0
+
+# A future 3.4 on its way
```

Comment:
Just to say I still prefer one of my other suggestions for this. Another suggestion "Look out for version 3.4 this Christmas"

Reply:
Even current version needs verb "a future version 3.4 is on its way" is least changing anything and has all the required elements of sentence. Unfortunately English is context averse so: "a future 3.4 on its way" has no reference to what "3.4" is or what it is doing (even "to be" needs to be there in order to be understandable).

Reply:
@elstoc: we already discussed that and you come back on it. Your suggestions doesn't respect the idea here.
@johnny-bit: if I understand you, add "**is** on its way" would be better. I will add that.

Reply:
@Nilvus - "is" is required. however that would still lack specifier for "3.4". As I mentioned English is context averse and there are no implicit connectors, so even "A future 3.4 is on its way" lacks information required by english-brain to make sense of sentence. some languages have tools to imply you mean version. english - not in the slightest. "A future [...]" reads as an adjective without connecting noun because "3.4" can't be noun in english. So you need noun before "3.4" due to structure of brain-rotting english language and in order to make it make sense you have couple to choose from: "version", "release", "edition" or even "darktable"...

Yeah - English sucks but if you leave "a future 3.4 is on its way", the monolingual english speakers (or multilingual speakers of languages which are contex-averse) will have "mind bump" when trying to read it :/

Reply:
That's clearly something I don't understand about english style by now. But ok, arguments are precise and clear. Thanks. So, "A future darktable 3.4 is on its way" should be ok? But "A future darktable 3.4..." seems to sounds hard. So maybe, just "darktable 3.4 is on its way" should even be better and simpler?

Reply:
i'd opt for "darktable 3.4 is on its way" :) :+1:

Reply:
And so, it will be that. Thanks.